tests/Aspire.Hosting.Azure.CosmosDB.Tests/Aspire.Hosting.Azure.CosmosDB.Tests.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Do we need a separate assembly? Why not just include these in `Aspire.Hosting.Azure.Tests`?

Reply:
I followed the test projects of the other Aspire components - like the PostgreSQL one as @sebastienros gave as an example.